library/alloc/src/collections/btree/node.rs

```diff
@@ -662,20 +662,13 @@ impl<'a, K: 'a, V: 'a> NodeRef<marker::Mut<'a>, K, V, marker::Internal> {
     /// Adds a key/value pair, and an edge to go to the left of that pair,
     /// to the beginning of the node.
     fn push_front(&mut self, key: K, val: V, edge: Root<K, V>) {
-        assert!(edge.height == self.height - 1);
-        assert!(self.len() < CAPACITY);
+        debug_assert!(edge.height == self.height - 1);
+        debug_assert!(self.len() < CAPACITY);
```

Comment:
Can we leave these as asserts? I'd like to avoid changing assert to debug_assert for now.

Reply:
So are you saying that about this internal node's push_front or also the leaf's push_front?

Reply:
Anywhere where we changed assert -> debug_assert.